docs/components/banners.md

```diff
@@ -0,0 +1,249 @@
+
+![Hero image of a banner showing a transaction error message](assets/banners/banners_hero.png)
+
+## Contents
+
+- [Contents](#contents)
+- [Using banners](#using-banners)
+- [Banners example](#banners-example)
+- [Theming banners](#theming-banners)
+
+## Using banners
+
+A banner displays an important, succinct message, and provides actions for users to address (or dismiss the banner). It requires a user action to be dismissed.
+
+Banners should be displayed at the top of the screen, below a top app bar. They’re persistent and nonmodal, allowing the user to either ignore them or interact with them at any time. Only one banner should be shown at a time.
+
+### Import banners
+
+Before you can use banners, you need to import the Material Components package for Flutter:
+
+```dart
+package:flutter/material.dart.
+```
+
+You need to be using a [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+### Making banners accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+### Anatomy and key properties
+
+![Banners anatomy diagram](assets/banners/banners_anatomy.png)
+
+Banners consist of the following:
+
+1. Supporting illustration (optional)
+2. Container
+3. Text
+4. Buttons
+
+
+#### Supporting illustration
+
+&nbsp;         | Property
+-------------- | ------------------------ 
+**Illustration** | `leading`
+
+#### Container
+
+&nbsp; | Property
+------ | --------- 
+**Color** | `backgroundColor` 
+
+#### Text
+
+&nbsp;         | Property
+-------------- | ------------------------ 
+**Text label** | `content`
+**Color**      | `style` on `content` when using a `Text`
+**Typography** | `style` on `content` when using a `Text`
+
+#### Buttons
+
+&nbsp;         | Property
+-------------- | ------------------------ 
+**Buttons** | `actions`
+
+
+## Banners example
+
+`MaterialBanner`
+- [Class definition](https://master-api.flutter.dev/flutter/material/MaterialBanner-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/banner.dart)
+- [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/banners/regular)
```

Comment:
FYI, Google style guide uses sentence case for titles ("Dartpad demo")

Reply:
ah good to know, fixed. Thanks